src/Control/Bind.purs

```diff
@@ -73,11 +73,11 @@ foreign import arrayBind :: forall a b. Array a -> (a -> Array b) -> Array b
 -- |
 -- | An example is the `Unit` type, since there is only one
 -- | possible value which can be returned.
-class Discard a where
-  discard :: forall f b. Bind f => f a -> (a -> f b) -> f b
+class BindDiscard a where
+  bind_ :: forall f b. Bind f => f a -> (a -> f b) -> f b
```

Comment:
I'm thinking about it in context of rebindable syntax, without the word `bind` in there it probably won't be too obvious what it relates to.